stingray/largememory.py

```diff
@@ -791,6 +793,7 @@ def _chunkLCSpec(data_path, spec_type, segment_size, norm, gti, power_type,
                 silent=silent,
                 large_data=False)
 
+        # REVIEW: Check if freq check is to be done this way
```

Comment:
Is the check for similar frequencies to be done this way? Is this logically correct?